### PR TITLE
Update package to 'dylan-tool'

### PR DESCRIPTION
**Pull request by fraya**
Update package to 'dylan-tool' and fix missing package infrastructure (README, tests, etc).

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```
Comment (cgay) [inline LICENSE.txt]: I'm a little wary of changing the licensing of old code. I see that Doug Auclair put "opyright: (c) 2002, LGPL" in the original commit so I wouldn't change it to the 2007 version of LGPL, which could be different.  It looks like LGPL-2.1 was in effect in 2002. Update to that instead?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+```dylan
+aif(big-long-calculation())
+  foo(it)
+else
+  bar()
+end;
+```
+
+### References
```
Comment (cgay) [inline README.md]: This should probably be a second level header. `##`

File: tests/anaphora-test.dylan [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  test test-aif-on-success-if;
+  test test-aif-on-success-if-else;
+  test test-aif-on-failure-if;
+  test test-aif-on-failure-if-else;
+end suite;
+
+define suite anaphora-test-suite ()
+  suite aif-test-suite;
+end suite;
+
```
Comment (cgay) [inline tests/anaphora-test.dylan:59]: You don't need to define any suites. These tests will run so quickly I can't imagine ever wanting to run only a subset of them.  But it's up to you.

File: tests/library.dylan [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  use testworks;
+  use anaphora;
+
+  export anaphora-tests;
+end library anaphora-test;
+
+define module anaphora-test
+  use common-dylan;
+  use common-extensions,
+    import: { integer-to-string };
```
Comment (cgay) [inline tests/library.dylan]: Everything in `common-extensions` is reexported from `common-dylan` so you shouldn't need this separate import.

**Review by cgay — COMMENTED** [review]
Looks good with a few minor changes. Nice to have tests and some doc!

File: LICENSE.txt (hunk 1 above)
Comment (fraya) [reply LICENSE.txt]: My license-foo is weak, I thought that this LGPL was the same. I'll change it.

File: README.md (hunk 2 above)
Comment (fraya) [reply README.md]: Ok.

File: tests/anaphora-test.dylan (hunk 3 above)
Comment (fraya) [reply tests/anaphora-test.dylan:59]: You are right. I love the ceremony.

File: tests/library.dylan (hunk 4 above)
Comment (fraya) [reply tests/library.dylan]: Thank you. I'll change it.

**Review by cgay — APPROVED** [review]

**Pull request merged**